File: clojure/raindrops/src/raindrops.clj
(ns raindrops)

(defn convert [number]
  (cond-> nil
    (zero? (rem number 3)) (str "Pling")
    (zero? (rem number 5)) (str "Plang")
    (zero? (rem number 7)) (str "Plong")
    :default (or (str number))))

File: solutions/csharp/raindrops/3/Raindrops.cs
using System;

public static class Raindrops
{
    public static string Convert(int number)
    {
        var rain = string.Empty;
        if (number % 3 == 0) rain += "Pling";
        if (number % 5 == 0) rain += "Plang";
        if (number % 7 == 0) rain +=  "Plong"; 
        if (string.IsNullOrEmpty(rain)) rain = number.ToString();
        return rain;
    }
}